src/main/kotlin/lotto/domain/Ticket.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.domain
+
+class Ticket(_numbers: List<Int>) {
+    val numbers = _numbers.toSet()
+
+    init {
+        require(numberIsInRange(TICKET_NUMBER_MIN, TICKET_NUMBER_MAX))
+        require(isSize(TICKET_NUMBER_SIZE))
```

Comment:
이미 상수로 정의되어 있어 파라미터로 넘기지 않고 아래와 같이 사용할수도 있을 것 같아요.

```suggestion
        require(numbers.size == TICKET_NUMBER_SIZE) { "로또의 개수는 6개 입니다."}
```